### PR TITLE
added initial .travis.yml

### PR DESCRIPTION
**Pull request by krichter722**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: c
+compiler:
+ - gcc
+ - clang
+before_install:
+ - sudo apt-get -qq update && sudo apt-get -qq build-dep git && sudo apt-get -qq install clang gcc
```
Comment (Shooter7119) [inline .travis.yml]: aQodJOsGpA

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+# `make check` is present, but does strange things, `make test` works; there's no `make distcheck` target
+script: autoreconf -v -i -f && ./configure && make -j24 && make -j24 test && sudo make install
```
Comment (Shooter7119) [inline .travis.yml]: aQodJOsGpA

**Comment by stefanbeller** [comment]
See https://submitgit.herokuapp.com/ or Documentation/SubmittingPatches

**Comment by sschuberth** [comment]
Also, by now there has been sent [another patch](http://thread.gmane.org/gmane.comp.version-control.git/278633) to add Travis CI support to the mailing list. Please join that thread on the list to take part in the review process.

**Comment by sschuberth** [comment]
Travis CI support now is [available in master](https://github.com/git/git/blob/master/.travis.yml), so this can be closed.

**Pull request closed without merging**